reason-healthcare/reason-framework
# Feature testing

- Fix selection behavior bug
- Add cpg-test-support to monorepo build
- Add default content, terminology, and cpg server endpoints for feature testing
- Add REST endpoint handling (in addition to file)
- Add version handling for PDs
- cpg-test-support package should not be a dependency of itself
- Fix communication request and immunization recommendation bug ($apply bug)
